### PR TITLE
Show call to action and edit button

### LINKED ISSUE #737: display edit button on the bottom card (if brand & name & nutriscore & nova are all empty)
display edit button on the bottom card (if brand & name & nutriscore & nova are all empty)

image

Add a hint that you can slide-up the card for more details. This is especially useful when it is empty.
```
if brand && name && nutriscore && nova is empty: display edit button
if nutriscore && nova is empty && misc =! nova-not-applicable && misc=! nutriscore-not-applicable: display edit button
```

### PR DESCRIPTION
**Pull request by aleene**
## PR Description
Show call to action and edit button, if the the product has missing data.

## Screenshot
image

## Type of Changes 
- [X] Fixes Issue #737

### CODE REVIEW

**Comment by teolemon** [comment]
Can you add a small screenshot @aleene ?

**Comment by aleene** [comment]
> Can you add a small screenshot

See the issue

File: Sources/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -304,6 +304,7 @@
 "call-to-action.summary" = "Take a photo of the front";
 "call-to-action.ingredients" = "Take a photo of the ingredients";
 "call-to-action.nutrition" = "Take a photo of the nutrition facts";
+"call-to-action.edit" = "Please edit this product.";
```
Comment (teolemon) [inline Sources/Localization/en.lproj/Localizable.strings]: I would personalize to get a clear view of the expected result.
Please do XYZ to get ABC

Comment (aleene) [reply Sources/Localization/en.lproj/Localizable.strings]: "Edit product details to have it analysed"?

Comment (teolemon) [reply Sources/Localization/en.lproj/Localizable.strings]: "Complete the product to get it scored" ?

**Review by tejuamirthi — DISMISSED** [review]

File: Sources/Localization/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -304,6 +304,7 @@
 "call-to-action.summary" = "Take a photo of the front";
 "call-to-action.ingredients" = "Take a photo of the ingredients";
 "call-to-action.nutrition" = "Take a photo of the nutrition facts";
+"call-to-action.edit" = "Complete the product to get it scored.";
```
Comment (teolemon) [inline Sources/Localization/en.lproj/Localizable.strings]:
```suggestion
"call-to-action.edit" = "Complete the product to get it scored";
```

**Review by teolemon — APPROVED** [review]

**Pull request merged**